docs/examples/bci2000/index.md

```diff
@@ -0,0 +1,24 @@
+# BCI2000
+
+BCI2000 is an open-source, all around purpose software designed for research with brain-computer interface (BCI).
+
+Let's go over how to connect and use your Wearable Sensing DSI headset with BCI2000.
+
+Please see the official BCI2000 page for how to install the latest version! [https://www.bci2000.org/mediawiki/index.php/DownloadBCI2000](https://www.bci2000.org/mediawiki/index.php/DownloadBCI2000)
```

Comment:
I'd mention that you will create a free account that can be used to download all modules and applications used by BCI2000.